src/Tizen.NUI/src/public/NUIWidgetApplication.cs

```diff
@@ -62,6 +62,30 @@ public NUIWidgetApplication(System.Type widgetType, string styleSheet) : base(ne
             core?.RegisterWidgetInfo(new Dictionary<System.Type, string> { { widgetType, ApplicationInfo.ApplicationId } });
         }
 
+        /// <summary>
+        /// Add WidgetInfo in runtime
+        /// </summary>
+        /// <param name="widgetType">Derived widget class type.</param>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
please remove <since_tizen> because it is hidden api. <since_tizen> is added when ACR is done. thank you.